src/app/window-handler.ts

```diff
@@ -1006,6 +1011,17 @@ export class WindowHandler {
         reloadWindow(focusedWindow as ICustomBrowserWindow);
     }
 
+    /**
+     * Zoom in
+     */
+    private onZoomIn(): void {
+        const focusedWindow = BrowserWindow.getFocusedWindow();
+        if (!focusedWindow || !windowExists(focusedWindow)) {
+            return;
+        }
+        focusedWindow.webContents.zoomLevel += 0.5;
```

Comment:
Use a named constant for this? And why 0.5 specifically?

Reply:
From Electron source code
zoomin: {
    label: 'Zoom In',
    accelerator: 'CommandOrControl+Plus',
    nonNativeMacOSRole: true,
    webContentsMethod: (webContents) => {
      webContents.zoomLevel += 0.5
    }
  },

So we zoom with same level as Command/+